### ISSUE TITLE
Base.active_project in Windows uses a lower case 'c' for the Drive in VSCode Repl

### ISSUE THREAD

**Issue by habemus-papadum** [julia-vscode/julia-vscode#3430]
Hi,
It looks like in Windows when the vscode extension launches julia for a specific environment, the `--project=<dir>` it uses in the launch invocation uses lower case for the drive name.
  
For instance, the `Base.load_path` from the VSCode repl looks like:
```
julia> Base.load_path()
3-element Vector{String}:
 "c:\\Users\\nehal\\src\\julia-fun\\warpsnwefts\\Project.toml"
 "C:\\Users\\nehal\\.julia\\environments\\v1.9\\Project.toml"
 "C:\\Users\\nehal\\.julia\\juliaup\\j" ⋯ 21 bytes ⋯ "mingw32\\share\\julia\\stdlib\\v1.9"
```

When julia is started manually using e.g. `julia --project=@` from WindowsTerminal, it results in a uppercase letter for the drive -- this casing is also consistent with the other entries in the `Base.load_path.`

This matters because `CondaPkg` triggers rebuilding of all Conda packages on any trivial use of the api once is recognizes  a difference the load_path it previously relied on vs the currently reported load path.  This means a lot of thrashing if your workflow switches between Julia launched from vscode and the normal Repl.  

While I am not a fan of the way CondaPkg decides to nuke conda environments without explicit permission, I think the proper fix is probably in vscode extension.

**Comment by pfitzseb** [comment]
While we could probably work around that, this really feels like an issue with CondaPkg. It shouldn't rely on string comparisons to determine file identity.

**Comment by habemus-papadum** [comment]
I agree, but it also makes sense to use capitalization consistent with effectively happens when `--project=@` is used. 

In any case it makes for a very strange user experience on Windows and given the vscode extension's excellent track record in offering a "just works" experience for users, maybe you could track this issue even it it ultimately needs resolution in `CondaPkg`

**Comment by habemus-papadum** [comment]
The relevant issue in CondaPkg is here: https://github.com/JuliaPy/CondaPkg.jl/issues/109
